src/libs/API/parameters/UpdateCustomRules.ts

```diff
@@ -0,0 +1,6 @@
+type UpdateCustomRules = {
+    policyID: string;
+    customRules: string;
```

Comment:
The API is expecting this. 
```suggestion
    description: string;
```